libafl/src/stages/logics.rs

```diff
@@ -280,12 +280,12 @@ where
         }
     }
 
-    fn restart_progress_should_run(&mut self, state: &mut Self::State) -> Result<bool, Error> {
```

Comment:
I wouldn't change this name, it might be confusing / people might (try to) call it in a non-restarting context, no?